esmvaltool_sample_data/sample_data.py

```diff
@@ -144,7 +161,13 @@ def search(connection, preferred_hosts, ignore_hosts, facets):
             datasets[dataset_name] = {}
         datasets[dataset_name][host] = dataset
 
-    print("Found", len(datasets), "unique datasets")
+    print(f"Found {len(datasets)} datasets")
```

Comment:
```suggestion
    print(f"Found {len(datasets)} datasets when using only the latest versions")
```
would be more clear, because on line 154 a very similar message is printed